docker-compose.yml

```diff
@@ -28,11 +28,7 @@ services:
     image: docker.io/linuxforhealth/jetstream:v0.0.19
     volumes:
       - ./local-config/nats:/conf
-    command: server --tls \
-      --tlscacert=/conf/lfh-root-ca.pem \
-      --tlscert=/conf/lfh-nats-server.pem \
-      --tlskey=/conf/lfh-nats-server.key \
-      --config=/conf/nats-server.conf
+    command: server --config=/conf/nats-server.conf
```

Comment:
moved all config to the conf file, since we were already using the conf file anyway.